### PR TITLE
added otp api

### PR DESCRIPTION
**Pull request by ankitdas13**
otpGenerate reference : https://razorpay.com/docs/payments/payment-gateway/s2s-integration/json/v2/build-integration/cards/#otp-generation-

otpSubmit reference : https://razorpay.com/docs/payments/payment-gateway/s2s-integration/json/v2/build-integration/cards/#response-on-submitting-otp

otpResend reference : https://razorpay.com/docs/payments/payment-methods/cards/authentication/native-otp/#otp-resend

### CODE REVIEW

File: test/fixtures/fake_otp_generate.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "url": "https://api.razorpay.com/v1/payments/pay_FVmAstJWfsD3SO/otp_resend/json?key_id=<YOUR_KEY_ID>"
+      }
+    ],
+    "metadata": {
+      "issuer": "HDFC",
+      "network": "MC",
+      "last4": "1111",
+      "iin": "411111"
+    }
+  }
```
Comment (sonyy-m) [inline test/fixtures/fake_otp_generate.json:19]: Fix indentation

File: test/fixtures/fake_otp_resend.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "next": [
+      "otp_submit",
+      "otp_resend"
+    ],
+    "razorpay_payment_id": "fake_payment_id"
+  }
```
Comment (sonyy-m) [inline test/fixtures/fake_otp_resend.json:7]: Fix indentation and add EOL char

File: test/fixtures/fake_otp_submit.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+ "razorpay_payment_id": "fake_payment_id",
+ "razorpay_order_id": "order_9A33XWu170gUtm",
+ "razorpay_signature": "9ef4dffbfd84f1318f6739a3ce19f9d85851857ae648f114332d8401e0949a3d"
+}
```
Comment (sonyy-m) [inline test/fixtures/fake_otp_submit.json:5]: End each file with a newline

File: documents/payment.md [hunk 4]
```diff
@@ -0,0 +1,632 @@
...
+paymentId = "pay_FVmAstJWfsD3SO";
+
+Razorpay::Payment.otp_generate(paymentId, para_attr)
+```
+**Parameters:**
+
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| paymentId*    | integer | Unique identifier of the payment                                               |
+| paymentId*    | integer | Unique identifier of the payment  |
```
Comment (sonyy-m) [inline documents/payment.md:577]: remove duplicate line

**Review by sonyy-m — CHANGES_REQUESTED** [review]

File: test/fixtures/fake_otp_generate.json (hunk 1 above)
Comment (ankitdas13) [reply test/fixtures/fake_otp_generate.json]: fixed

File: test/fixtures/fake_otp_resend.json (hunk 2 above)
Comment (ankitdas13) [reply test/fixtures/fake_otp_resend.json]: fixed

File: test/fixtures/fake_otp_submit.json (hunk 3 above)
Comment (ankitdas13) [reply test/fixtures/fake_otp_submit.json]: fixed

File: documents/payment.md (hunk 4 above)
Comment (ankitdas13) [reply documents/payment.md]: removed duplicate payment id

**Review by sonyy-m — APPROVED** [review]
LGTM

**Pull request merged**